charts/datree-admission-webhook/templates/namespace-post-install.yaml

```diff
@@ -5,9 +5,10 @@ metadata:
   namespace: {{template "datree.namespace" .}}
   labels: {{include "datree.labels" . | nindent 4}}
   annotations:
+    argocd.argoproj.io/hook: PostSync
     "helm.sh/hook": post-install, post-upgrade
     "helm.sh/hook-weight": "-5"
-    "helm.sh/hook-delete-policy": hook-succeeded, hook-failed
+    "helm.sh/hook-delete-policy": before-hook-creation, hook-succeeded, hook-failed
```

Comment:
argocd recommends to add `before-hook-creation` for `post-upgrade`
https://argo-cd.readthedocs.io/en/stable/user-guide/helm/

Reply:
I saw in the link references to `pre-upgrade` or `post-upgrade` regarding the `before-hook-creation`.
In what way will this change the installation process?

Reply:
it will delete and recreate if needed